services/daily.py

```diff
@@ -6,7 +6,11 @@
 from etl.glams_table import close_glams_connections, get_active_glams, load_glams_images, open_glams_connections, refresh_glams_visualizations
 from etl.etl_glam import process_glam
 from etl.download_mediacounts import download_file
+import datetime
 
+with open("cronjobDaily.txt", mode='a') as file:
```

Comment:
do we need it in prod?

Reply:
It doesn't must, but for the order and so that the user can make sure that it works, I added the above code that will write in the cronjobDaily.txt file the time when the cron is activated